### PR TITLE
Shopify Admin Store Connector

### PR DESCRIPTION
**Pull request by thatapiguy**
---
When submitting a connector, please check the following conditions for your PR.

- [x] I attest that the connector works and I verified by deploying and testing all the operations.
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a logo.

### CODE REVIEW

**Comment by thatapiguy** [comment]
certify-connector

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: independent-publisher-connectors/Shopify Admin Store/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,2369 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0",
+    "title": "Shopify Admin Store",
+    "description": "Manage a shopify store's customers, products, collections and customers using the Shopify Admin API.",
+    "contact": {}
```
[the remark this answers is not in the document]
Comment (thatapiguy) [reply independent-publisher-connectors/Shopify Admin Store/apiDefinition.swagger.json]: Added contact info

**Review by mamurshe — CHANGES_REQUESTED** [review]
Please remove open weather

**Comment by thatapiguy** [comment]
> Please remove open weather

I deleted them from my branch and then committed the change.  Not sure why they are still there.

**Comment by natalie-pienkowska** [comment]
@mamurshe will resubmit the PR so that it doesn't delete the OpenWeather connector and then @sriyen-msft will be able to publish for certification.

**Comment by Amjed-Ayoub** [comment]
Dear partner,

We apologize for the delay you might be experiencing while we are working on a high volume of connector requests. Rest assured that we are doing everything we can to address this volume. We are taking measures like improving our processes and hiring additional team members to guarantee resolving your connector requests in a timely manner. 

Please let us know if you have any questions in the meantime.

File: independent-publisher-connectors/Shopify Admin Store/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Shopify Admin Store
+Manage a shopify store's customers, products, collections and customers using the Shopify Admin API . [Shopify Online Store API](https://shopify.dev/docs/admin-api/rest/reference/online-store)
+
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Shopify Admin Store/README.md:3]: @thatapiguy Please refer the templates and update your readme.md with all the required sections

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md.

File: independent-publisher-connectors/Shopify Admin Store/README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+## Pre-requisites
+To use this connector, you need the following
+1. A Microsoft Power Apps or Power Automate plan with custom connector feature
+2. A Shopify store URL (e.g. storename.myshopify.com )
+3. Shopify client credentials - client id and client secret 
+
+###### How to get Client Credenitals
+* [Sign Up](https://partners.shopify.com/signup) for Shopify Partner Account.
+* [Log In](https://partners.shopify.com/organizations) with your account details and generate credentials by creating an app using: [Generate credentials from your partner dashboard](https://shopify.dev/tutorials/authenticate-a-public-app-with-oauth#generate-credentials-from-your-partner-dashboard)
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Shopify Admin Store/README.md:13]: @thatapiguy Can you please provide all the instructions in step-by-step with screenshots that can be understandable by everyone?

File: independent-publisher-connectors/Shopify Admin Store/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,2373 @@
...
+            "in": "body",
+            "required": true,
+            "description": "Order details",
+            "schema": {
+              "$ref": "#/definitions/CreateapendingorderwithanexistingcustomerRequest"
+            }
+          }
+        ],
+        "responses": {
+          "default": {
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Shopify Admin Store/apiDefinition.swagger.json:52]: @thatapiguy Default responses are not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid expected response codes with schema definitions for all the operations in the swagger. **This applies to all the default responses in the swagger.**

File: independent-publisher-connectors/Shopify Admin Store/apiProperties.json [hunk 5]
```diff
@@ -0,0 +1,151 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2generic",
+          "clientId": "",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Shopify Admin Store/apiProperties.json:8]: @thatapiguy Please provide the value as `"[[Provide your client id]]"` or `"[[TEST]]"` or "[[DUMMY]]"`

File: independent-publisher-connectors/Shopify Admin Store/apiProperties.json [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+          "scopes": [
+            "read_analytics",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Shopify Admin Store/apiProperties.json:10]: @thatapiguy Please make sure that you are documenting all these scores in the readme.md under obtaining credentials section.

File: independent-publisher-connectors/Shopify Admin Store/apiProperties.json [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+          "constraints": {
+            "required": "true"
+          },
+          "description": "Specify the API Version. eg. 2021-04",
+          "displayName": "Shopify API Version. eg. 2021-04",
+          "tooltip": "Provide your Shopify API Version"
+        }
+      }
+    },
+    "iconBrandColor": "#64943E",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Shopify Admin Store/apiProperties.json:82]: @thatapiguy Please refer the template for apiProperties.json file and change it to default iconBrandColor

**Review by sriyen-msft — CHANGES_REQUESTED** [review]
@thatapiguy Please do not remove or edit other connector files in your PR.

**Comment by sriyen-msft** [comment]
@thatapiguy I'm just checking in to see how things are going with updating your connector artifacts as per the given feedback. Please feel free to reach out with any questions you may have, Thanks.

**Comment by Amjed-Ayoub** [comment]
Hello @thatapiguy,
I am sending this to follow up with you about getting your request approved.
Please let me know if you have any questions.
Thank you very much

**Comment by Amjed-Ayoub** [comment]
Hello @thatapiguy, I hope you are doing well. 
I am following up with you to see how things are going with updating your connector. 
Please do not hesitate to reach out if you have any questions.
Thank you.

**Pull request closed without merging**